packages/now-cli/test/dev/integration.js

```diff
@@ -319,6 +319,53 @@ test(
   })
 );
 
+test(
+  '[now dev] handles miss after route',
+  testFixtureStdio('handle-miss-after-route', async (t, port) => {
+    const response = await fetchWithRetry(`http://localhost:${port}/post`);
+
+    const test = response.headers.get('test');
+    const override = response.headers.get('override');
+    t.is(test, '1', 'exected miss header to be added');
+    t.is(override, 'one', 'exected override header to not override');
+
+    const body = await response.text();
+    t.regex(body, /Blog/gm);
+  })
+);
+
+test(
+  '[now dev] handles miss after rewrite',
+  testFixtureStdio('handle-miss-after-rewrite', async (t, port) => {
```

Comment:
Let's add another test for this fixture:
`/about.html` returns 200 without any added headers
`/blog/post/one.html` returns 200 without any added headers
`/blog/post` returns 200 with headers: `test: 1` and `override: 2`

Reply:
Thanks, done 👍